openapi-python-templates/requirements.mustache

```diff
@@ -0,0 +1,4 @@
+pydantic >= 1.0
```

Comment:
I'm not sure about specific versions, and in `setup.py` I didn't fix them at all. If you know minimal versions for code generated client, please, tell me and I'll change it.

Reply:
I think I made this change, pydantic introduced some backward incompatible changes so in order to update the rest of the dependencies this was required, have a look at my previous MR.